test/OpenTelemetry.Exporter.Console.Tests/ConsoleActivityExporterTest.cs

```diff
@@ -0,0 +1,75 @@
+// <copyright file="ConsoleActivityExporterTest.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using OpenTelemetry.Trace;
+using Xunit;
+
+namespace OpenTelemetry.Exporter.Console.Tests;
+
+public class ConsoleActivityExporterTest
+{
+    /// <summary>
+    /// Test case for https://github.com/open-telemetry/opentelemetry-dotnet/issues/3863.
+    /// </summary>
+    [Fact]
+    public void VerifyConsoleActivityExporterDoesntFailWithoutActivityLinkTags()
+    {
+        var uniqueTestId = Guid.NewGuid();
+        var activitySourceName = $"activitySourceName{uniqueTestId}";
```

Comment:
What's the common practice?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/943c856c1c6295a536bc8a1c89896febc0be6107/test/OpenTelemetry.Exporter.Prometheus.HttpListener.Tests/PrometheusSerializerTests.cs#L34

Reply:
There's not a clear common practice.
In this repo I can see nameof(TestMethod), nameof(TestClass), and arbitrary strings used.
The Guid will give the strongest guarantee against duplicates and is safe from future copy/paste reusing this code block.

Reply:
Do you think it makes sense to ask for convergence instead of creating yet another way?

Reply:
> The Guid will give the strongest guarantee against duplicates and is safe from future copy/paste reusing this code block.

Could you see what's the downside?

Reply:
TBH I haven't spent time on this, some negative outcomes I can see are that a randomly generated GUID could be 1) very random, making it hard to replay 2) not following the normal naming convention, making it anti-pattern

Reply:
My suggestion is that if you want to take an approach, think through the pros/cons.